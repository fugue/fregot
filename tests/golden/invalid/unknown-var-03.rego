package fregot.tests.invalid.unkown_var_03

double(x) = y {
  y = x + x
}

test_double {
  double(x, y)
  y == 2
}
