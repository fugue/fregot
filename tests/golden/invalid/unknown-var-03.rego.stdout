passed: 1, failed: 0, errored: 0
